Servers/manage-logs.sh

```
@@ -0,0 +1,204 @@
+    if [ ! -d "$tenant_dir" ]; then
+        echo "Tenant '$tenant_id' not found."
+        return 1
+    fi
+
+    echo "=== Logs for tenant: $tenant_id ==="
+    echo "Location: $tenant_dir"
+    echo "Files:"
+    ls -la "$tenant_dir"
+    echo ""
+    echo "=== Latest log entries ==="
+    local latest_log=$(ls -t "$tenant_dir"/*.log 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)